src/main/java/Duke/Tasks/Task.java

```diff
@@ -0,0 +1,84 @@
+package duke.tasks;
+
+import duke.errors.InvalidCommandException;
+
+/**
+ * The type Task.
+ */
+public abstract class Task {
+    /**
+     * The Description.
+     */
+    protected String description;
```

Comment:
Since you have already had a getter method for description, maybe you can make it a private variable here? If you want to also modify it, I think a setter method may be helpful.